README.md

```diff
@@ -1,2 +1,149 @@
-# Yoga-Class-Refactored
-A Refactored Yoga Application
+# About
+- Backend application demo using Spring Boot
+- Application to manage Yoga classes
+- Refactored of [old version](https://github.com/web-development-course-way/Yoga-Class)
+# Tech Stack
+- Docker
+- PostgreSQL & pgAdmin 
+- Java & Spring boot
```

Comment:
only spring boot? can you list all dependencies? actuator/jpa/web...etc?

Reply:
Yes for sure, but is this considered as Technology to be listed?

Reply:
it should, if i want to google the technology stack, what will i google? just spring boot? spring boot contains multiple projects, so list the projects you used.

Reply:
Done